packages/visualizations/addon/components/number-format-dropdown.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *  {{number-format-dropdown
+ *    column=column
+ *    onUpdateFormat = (action 'onUpdateFormat')
+ *  }}
+ */
+
+import Component from '@ember/component';
+import layout from '../templates/components/number-format-dropdown';
+import { assign } from '@ember/polyfills';
+import { getWithDefault, set } from '@ember/object';
+
+export default Component.extend({
+  layout,
+
+  /**
+   * @property {Array} classNames
+   */
+  classNames: ['number-format-dropdown'],
+
+  actions: {
+    /**
+     * @action updateColumnNumberFormat
+     */
+    updateColumnNumberFormat() {
+      let { onUpdateReport, column } = this.attrs,
+          format = getWithDefault(this, 'format', column.format);
+
+      onUpdateReport('updateColumn', assign({}, column, { format }));
+    },
+
+    /**
+     * @action onUpdateFormat
+     */
+    onUpdateFormat({ format }) {
```

Comment:
If the `number-format-selector` is only going to update the format, do we need to wrap `format` in an object?

Reply:
object is required for metric label config

Reply:
`{{number-format-selector format=options.format onUpdateFormat=onUpdateConfig}}` metric label is passing in `onUpdateConfig` directly. If we want to wrap it up, we need another action in metric label. 

IMHO, returning object is more flexible than format string

Reply:
If there is a need for an object then it would be alright, but I'm not sure there is a need for it. 
Can we discuss this?

Reply:
OK. I will make it pass string instead of object.